baggage/baggage_test.go

```diff
@@ -152,14 +152,11 @@ func TestNewKeyValueProperty(t *testing.T) {
 
 	p, err = NewKeyValueProperty("key", "value")
 	assert.NoError(t, err)
-	assert.Equal(t, Property{key: "key", value: "value", hasValue: true, hasData: true}, p)
+	assert.Equal(t, Property{key: "key", value: "value", hasValue: true}, p)
 }
 
 func TestPropertyValidate(t *testing.T) {
 	p := Property{}
-	assert.ErrorIs(t, p.validate(), errInvalidProperty)
```

Comment:
I find `invalid baggage list-member property` a better error message than `invalid key`.

Reply:
It would be if this was baggage.validate(). Property.validate is a lower level, it doesn't know about baggage.

Reply:
That's not in scope of this PR. `member.validate` should be wrapping lower-level errors if it wants to return "invalid member" as top level error.

Reply:
Gotcha 👍